src/wasm-interpreter.h

```diff
@@ -171,7 +173,7 @@ class ExpressionRunner : public OverriddenVisitor<SubType, Flow> {
       if (flow.breaking()) {
         return flow;
       }
-      NOTE_EVAL1(flow.values);
+      NOTE_EVAL1(flow.getValues());
```

Comment:
Hmm, there are more uses of `flow.values` in this file, but actually I'm not very sure at this point, because some of the uses are not gets but sets, so to make this really consistent, we should also make `setValues()` or something, but I think at this point it is an overkill... This is actually one of the reasons that I was not sure about renaming `Flow::getSingleValue`. 

I suggest we either don't rename `Flow::getSingleValue` or just revert this `getValues()` change. I'm sorry for flip-flopping ;(

Reply:
np. Just reverted last commit